spec-main/api-browser-window-spec.ts

```diff
@@ -1005,6 +1005,24 @@ describe('BrowserWindow module', () => {
           await restore;
           expectBoundsEqual(w.getNormalBounds(), bounds);
         });
+        it('does not change size for a frameless window', async () => {
+          w.destroy();
+          w = new BrowserWindow({
+            show: false,
+            frame: false,
+            width: 300,
+            height: 300
```

Comment:
I was only able to repro when the window had `minWidth` and `minHeight` set, were you able to repro without it ?

Reply:
I have updated the test to set min size, they are required for reproducing the bug.